src/main/java/net/szum123321/textile_backup/core/create/BackupHelper.java

```diff
@@ -37,44 +39,31 @@
 	private final static ConfigHelper config = ConfigHelper.INSTANCE;
 
 	public static Runnable create(BackupContext ctx) {
-		if(config.get().broadcastBackupStart) {
-			Utilities.notifyPlayers(ctx.getServer(),
-					ctx.getInitiatorUUID(),
-					"Warning! Server backup will begin shortly. You may experience some lag."
-			);
-		} else {
-			log.sendInfoAL(ctx, "Warning! Server backup will begin shortly. You may experience some lag.");
-		}
+		int delay = 0;
```

Comment:
Set delay to always equal the configured value. By using it only when announcing the backup you create an ambiguous behaviour.

Reply:
I thought of this aswell but I thought it would be rather weird to have a delay that is invisible which might be even more confusing. Hence I named the variable to be an additional parameter to the broadcast. 

But I'll do so if you want